### PR TITLE
more info about Azure Pipelines

### PR DESCRIPTION
**Pull request by garyverhaegen-da**

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -106,3 +106,97 @@ Therefore, the vast majority of releases are made using a target commit from a
...
+On the build page view (when no specific job or step is selected), you can see
+the build artifacts. Most of these artifacts are additional logs, presumably
+more detailed.
+
+### Managing jobs in Azure Pipeleines
+
+Only a few people have access to Azure Pipelines directly. Those people can
+additionally use the Azure Pipelines UI to:
+
+- Cancel a running build. Note that we cannot cancel individual jobs, and the
```
Comment (yves-da) [inline ci/README.md:164]: Question: I thought that I can, without special privileges, cancel a running build by closing a PR on GitHub?

File: ci/README.md [hunk 2]
```diff
@@ -106,3 +106,97 @@ Therefore, the vast majority of releases are made using a target commit from a
...
+same code. This is particularly problematic for the platform-independence test,
+in rare cases where the various platform jobs don't start at the same time and
+the changes on main in-between change the produced DAR file.
+
+### Restarting a failed build
+
+Azure Pipelines should trigger a build on every pull request (but not every
+branch). If a build has failed and you believe the failure to be flaky, you can
+re-run the build by navigating to the "Checks" tab of your PR, and clicking the
+"Re-run failed checks" button in the top right.
```
Comment (yves-da) [inline ci/README.md:130]: Maybe also worthwhile to mention the `/azp run` commenting on a PR to re-run the build?

**Review by yves-da — APPROVED** [review]
Thanks a lot for the additional info on Azure. 🎉 

I added two small comments where something more could be said.

**Review by paulbrauner-da — APPROVED** [review]
Thank you!

File: ci/README.md (hunk 2 above)
Comment (garyverhaegen-da) [reply ci/README.md:130]: Oh right, thanks! I'll add that.

**Pull request merged**